omod/src/main/webapp/pages/labs/labPatientList.gsp

```diff
@@ -0,0 +1,209 @@
+
+        jq.get(openmrsContextPath + "/ws/rest/v1/pihapps/config?v=custom:(" + rep + ")", function(pihAppsConfig) {
+
+            const dateFormat = pihAppsConfig.dateFormat ?? "DD-MMM-YYYY";
+            const dateTimeFormat = pihAppsConfig.dateTimeFormat ?? "DD-MMM-YYYY HH:mm";
+            const primaryIdentifierType = pihAppsConfig.primaryIdentifierType?.uuid ?? '';
+            const conceptUtils = new PihAppsConceptUtils(jq);
+            const patientUtils = new PihAppsPatientUtils(jq);
+            const dateUtils = new PihAppsDateUtils(moment);
+            const orderFulfillmentStatusOptions = pihAppsConfig.labOrderConfig.orderFulfillmentStatusOptions;
+
+            orderFulfillmentStatusOptions.forEach((statusOption) => {
+                if (statusOption.status) {
+                    const option = jq("<option>").attr("value", statusOption.status).html(statusOption.display);
+                    jq("#orderFulfillmentStatus-filter").append(option);
+                }
+            });
+            jq("#orderFulfillmentStatus-filter").val(orderFulfillmentStatus);
+
+            // Column functions
+            const getEmrId = (patientWithOrders) => { return patientUtils.getPreferredIdentifier(patientWithOrders.patient, primaryIdentifierType); };
+            const getPatientName = (patientWithOrders) => { return patientWithOrders.patient.person.display; }
+            const getOrderDates = (patientWithOrders) => {
+                const orderDates = patientWithOrders.orders.map((order) => order.dateActivated).sort().reverse();
+                return orderDates.map((d) => dateUtils.formatDateWithTimeIfPresent(d, dateFormat, dateTimeFormat)).join(", <br/>");
+            }
+            const getLabTest = function(order) {
+                const urgency = order.urgency === 'STAT' ? '<i class="fas fa-fw fa-exclamation" style="color: red;"></i>' : '';
+                return urgency + conceptUtils.getConceptShortName(order.concept, window.sessionContext?.locale);
+            }
+            const getTestNames = (patientWithOrders) => {return patientWithOrders.orders.map((o) => getLabTest(o)).join(", <br/>");};
+
+            const getFilterParameterValues = function() {
+                return {
+                    "orderType": pihAppsConfig.labOrderConfig.labTestOrderType?.uuid,
+                    "patient": jq("#patient-filter-field").val(),
+                    "labTest": jq("#testConcept-filter").val(),
+                    "activatedOnOrAfter": jq("#orderedFrom-filter-field").val(),
+                    "activatedOnOrBefore": jq("#orderedTo-filter-field").val(),
+                    "orderFulfillmentStatus": jq("#orderFulfillmentStatus-filter").val(),
+                    "sortBy": "dateActivated-desc"  // TODO: Sorting by dateActivated desc does not seem right, but doing this to match existing labWorkflow, but shouldn't this order by urgency and asc?
```

Comment:
Seems reasonable... I guess the idea with "desc" is if there was a lot of old data you didn't want it clogging the top?